.gitignore

```diff
@@ -0,0 +1,88 @@
+# This file is used to ignore files which are generated
```

Comment:
this file contains a bunch of stuff that should not exist in this project, I think that is not a good idea to have it.